bip-encrypted-backup.md

```diff
@@ -0,0 +1,268 @@
+```
+BIP: ?  
+Layer: Applications  
+Title: Bitcoin Encrypted Backup  
+Author: // TBD  
+Comments-Summary: No comments yet.  
+Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-????  
+Status: Draft  
+Type: Informational  
+Created: 2025-08-22  
+License: BSD-2-Clause  
+```
+
+## Introduction
+
+### Abstract
+
+This BIP defines a compact encryption scheme for **wallet descriptors** (BIP-0380),
+**wallet policies** (BIP-0388), **labels** (BIP-0329), and
+**wallet backup metadata** (json). The encrypted output—called a
+*bitcoin encrypted backup* (BEB), enables users to outsource long‑term storage to
```

Comment:
This scheme is generic enough that I don't think it needs the word "bitcoin" in it. I also don't think we need to pick a "marketing name", something more boring like "BIP_XXX encrypted backup" should be fine.